projects/archiver/media.ts

```diff
@@ -0,0 +1,24 @@
+import { Front, ImageElement, CAPIArticle, Image } from './common'
+export const getImagesFromArticle = (article: CAPIArticle): Image[] => {
+    const image = article.image
+    const elements = article.type === 'article' ? article.elements : []
+    const images = elements
+        .filter((element): element is ImageElement => element.id === 'image')
+        .map(element => element.src)
+
+    return image ? [image, ...images] : images
+}
+
+export const getImagesFromFront = (front: Front): Image[] => {
+    let articles: CAPIArticle[] = []
+    front.collections.forEach(front => {
```

Comment:
Any way this can be a little less nested?

Reply:
🐦 done.